.clang-tidy

```diff
@@ -11,9 +12,12 @@ Checks:
     cert-env33-c,
     cert-err34-c,
     cert-err*-cpp,
+    -cert-err58-cpp,
     cert-flp30-c,
     cert-oop*,
     cppcoreguidelines-*,
+    -cppcoreguidelines-avoid-magic-numbers,
```

Comment:
I don't think we should disable this check entirely. I think it should be demoted to a warning instead. Thoughts?

Reply:
Same here. I meant to demote it. Will update.

Reply:
Done.